daffodil-lib/src/main/resources/IBMdefined/GeneralPurposeFormat.xsd

```diff
@@ -43,7 +43,7 @@
 * <xs:schema xmlns:tns="http://www.ogf.org/dfdl/dfdl-1.0/examples/" ... >
 *
 * <xs:import namespace="http://www.ogf.org/dfdl/dfdl-1.0/examples/"
-*  schemaLocation="DFDLGeneralFormatDefinition.xsd"/>
+*  schemaLocation="DFDLGeneralFormatDefinition.dfdl.xsd"/>
```

Comment:
This change is not correct for this file. I think this file wants to be reverted back to what it was.

btw: do we get to remove IBM's copyright on this file, or do we have to keep it and add the ASF license block?

Reply:
Will revert changes to this file. I think because IBM signed the SGA it's correct to use just the Apache license block?